README.md

```diff
@@ -1 +1,47 @@
-# Awesome_books
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+
+# Awesome books: plain JavaScript with objects
+
+> This project aimed at setting up a mobile and desktop version of my personal portfolio and the following objectives were expected:
+
+- UnUnderstand different ways to create objects in JavaScript.
+- Create and access properties and methods of JavaScript objects.
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML, JavaScript
+
+## Getting Started
+
+**Requirements**
+
+- HTML, JavaScript linters
+- Github Pages
+
+
```

Comment:
[OPTIONAL]

- I recommend to add also how to clone and run the project in local. Think about the user who doesn't have technical knowledge.

Reply:
done